src/components/component.ts

```diff
@@ -296,6 +296,10 @@ export class Component {
     return this;
   }
 
+  public renderLowPriority() {
+    return this.render();
```

Comment:
What does low priority mean? I haven't seen any docs about it.

Reply:
It's what you call instead of render to use the deferred rendering codepath (when available). When writing the deferred renderer originally, i didn't want to mess with the existing uses of `.render()` so i introduced `.renderLowPriority` on `Plottable.Plot` which triggers the deferred callbacks. This change just pushes the method up the `Plottable.Component`

Reply:
Hmm, but deferred rendering only exists on Plot, so I worry users will get confused by the difference (especially since there's no discernible distinction).